Source/PeleLMeX_DefaultIBC.H

```diff
@@ -0,0 +1,178 @@
+// Problem-specific Default Initial Boundary Conditions struct (IBC)
+#ifndef PROBLEM_IBC_H
+#define PROBLEM_IBC_H
+
+#include <AMReX_Geometry.H>
+#include <AMReX_FArrayBox.H>
+#include <AMReX_REAL.H>
+
+#include <PMFData.H>
+
+struct ProbParmDefault
+{
+  /**
+   * PeleLMeX always needs a mean thermodynamic pressure
+   */
+  amrex::Real P_mean = 101325.0;
+};
+
+struct DefaultProbIBC
+{
+  /**
+   * \brief Initialize data at the beginning of the simulation
+   *
+   * \param i                 i index.
+   * \param j                 j index.
+   * \param k                 k index.
+   * \param is_incompressible incompressible flag.
+   * \param state             state Array4.
+   * \param aux               auxiliary Array4.
+   * \param geomdata          geometry data.
+   * \param prob_parm         problem specific parameters.
+   * \param pmf_data          PMF (Premixed Flame) data
+   */
+  AMREX_GPU_DEVICE
+  AMREX_FORCE_INLINE
+  static void initdata(
+    int /*i*/,
+    int /*j*/,
+    int /*k*/,
+    int /*is_incompressible*/,
+    amrex::Array4<amrex::Real> const& /*state*/,
+    amrex::Array4<amrex::Real> const& /*aux*/,
+    amrex::GeometryData const& /*geomdata*/,
+    ProbParmDefault const& /*prob_parm*/,
+    pele::physics::PMF::PmfData::DataContainer const* /*pmf_data*/)
+  {
+    /** Notes:
```

Comment:
Add an amrex::Abort() here so if the function gets called without being overridden it doesn't cause unexpected behavior for the user